src/streaming/streamingclient.py

```diff
@@ -0,0 +1,144 @@
+"""StreamingClient tool used for streaming services"""
+from src.streaming.MediaConfig import MediaConfig
+import websocket
```

Comment:
You'll need to add this to requirements.txt

Reply:
do you mean the `__index__.py` file ?

Reply:
I think the MediaConfig file did not get added to git

Reply:
We don't use an `__index__.py` file. pip only requires a `requirements.txt` file with requirements and their versions